patches/components-search_engines-generated_marketing_snippets.cc.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/components/search_engines/generated_marketing_snippets.cc b/components/search_engines/generated_marketing_snippets.cc
+index c08eb99eeadeb4bf54a1f2c898722c58177119ac..49ebd74438680d165310c7e8281265928dec3e00 100644
+--- a/components/search_engines/generated_marketing_snippets.cc
++++ b/components/search_engines/generated_marketing_snippets.cc
+@@ -21,10 +21,6 @@ int GetMarketingSnippetResourceId(const std::u16string& engine_keyword) {
+       base::WideToUTF16(TemplateURLPrepopulateData::brave.keyword)) {
+     return IDS_BRAVE_SEARCH_DESCRIPTION;
+   }
+-  if (engine_keyword ==
```

Comment:
can we redefine duckduckgo and qwant to be something that will never match anything here? Alternatively we maybe could do something like `#define duckduckgo duckduckgo.keyword) &&  "" == base::WideToUTF16(TemplateURLPrepopulateData::duckduckgo`

Reply:
Addressed in 360ddb7e8875c68e955719ace5c1a627944590d6. We don't need to patch these out, just need to include our own redefinitions of those engines.